### ISSUE TITLE
Can not share to WordPress from Apple Notes

### ISSUE THREAD

**Issue by charliescheer** [wordpress-mobile/WordPress-iOS#12008]
### Expected behavior
If you go to apple notes, and click on the share button you should be able to select to share to the WordPress app, and share as a draft to the WordPress app.

### Actual behavior
In WordPress 12.6 and above WordPress does not appear as an option to share to.  I tested this on a few other apps and it seems to be isolated to apple notes.

<img src="https://cdn-std.dprcdn.net/files/acc_593859/zFHiDP" />

@jkmassel tested this with me and he was not seeing the same problem when using iOS 12.3, so it may be an iOS problem as well.

### Steps to reproduce the behavior
On a device that is an older version of iOS than 12.3:

1. open apple notes
2. click on the sharing button
3. click on more to look for WordPress.

WordPress 12.6, 12.6.1  (also tested on 12.5.1 and it was working fine)
iOS 12.3.1

**Comment by frosty** [comment]
@nheagy You mentioned that we made a change to the sharing predicate around this time?

**Comment by nheagy** [comment]
Yes, here: https://github.com/wordpress-mobile/WordPress-iOS/pull/11780

It's tough to wrap my head around. The change was just supposed to remove the limit for how many images could be shared. iOS 12 and 13 may be different because the format(s) notes shares in may have changed.

**Comment by synora** [comment]
We have a user in 3156719zen reporting this same issue.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by kean** [comment]
**RCA:** these queries need to be updated from `@count == 1` to `@count >= 1`

```
                SUBQUERY(
                $extensionItem.attachments,
                $attachment,
                ANY $attachment.registeredTypeIdentifiers UTI-CONFORMS-TO "public.plain-text"
                || ANY $attachment.registeredTypeIdentifiers UTI-CONFORMS-TO "public.text"
                ).@count >= 1
```

Noting that the editor used in the extension doesn't currently fully support rich text.
